app/src/test/java/com/vero/tasky/auth/domain/usecase/password/ValidatePasswordUseCaseTest.kt

```diff
@@ -0,0 +1,30 @@
+package com.vero.tasky.auth.domain.usecase.password
+
+import com.google.common.truth.Truth.assertThat
+import org.junit.Test
+
+class ValidatePasswordUseCaseTest {
```

Comment:
As mentioned on my previous comment

What is a password is "ABCDEFGHIJKLM"
It will return True
When according to the client´s expectations (Tasky´s usecase) it should return False